### ISSUE TITLE
any killable and enemy hp argument doesn't not seem to be working.

### ISSUE THREAD

**Issue by EisiaDEV** [BananaPuncher714/LethalQuantities#106]
![image](https://github.com/user-attachments/assets/a43ca236-62f0-426c-be6b-34827128ac5d)
jester won't be killed or damaged, only can be stunned.

**Comment by EisiaDEV** [comment]
All other mobs that were previously invincible are also expected to be applicable.

**Comment by BananaPuncher714** [comment]
Yes, that's not something which this mod can change, since there is custom health damage handling for every enemy. This can only makes it so killable enemies are unkillable.
